determined-ai/determined
# fix: rename `last_login` to `last_auth_at`

## Description
Rename `last_login` column to `last_auth_at` for more accuracy




## Test Plan
- Check if these APIs work
  - api/v1/users should return users with `last_auth_at`
  - api/v1/user should return the current user with `last_auth_at`
- Check if `Last Seen` column display the data in user management table 




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
